dts/riscv/andes/andes_v5_ae350.dtsi

```diff
@@ -122,7 +192,9 @@
 			riscv,max-priority = <255>;
 			riscv,ndev = <1023>;
 			interrupts-extended = <&CPU0_intc 3 &CPU1_intc 3
-					       &CPU2_intc 3 &CPU3_intc 3>;
```

Comment:
Why did the irq number change only for CPU2 and CPU3, and not for CPU0 and CPU1?

Reply:
Sorry, it is our mistake.
The number 11 for  is not correct, we have fixed it.
All the cpu interrupt number in plic-sw need to be 3.